go.mod

```
@@ -10,6 +10,8 @@ tool (
 	golang.org/x/tools/cmd/stringer
 )
 
+replace github.com/minio/pkg/v3 => ../pkg
```

replace with your fork so tests can run. To reference https://github.com/minio/pkg/pull/184

```suggestion
replace github.com/minio/pkg/v3 => github.com/0xMALVEE/pkg/v3 bdf1872b5b0181499826a2fc326a1dfb607cef75
```

.. (the last is the commit hash of the [latest commit](https://github.com/0xMALVEE/pkg/commit/bdf1872b5b0181499826a2fc326a1dfb607cef75) on [your branch](https://github.com/0xMALVEE/pkg/tree/new-key-s3-content-type).